rules/do-you-use-co-creation-patterns/rule.md

```diff
@@ -0,0 +1,112 @@
+---
+type: rule
+title: Do you use Co-Creation Patterns?
+uri: do-you-use-co-creation-patterns
+authors:
+  - title: Daniel Mackay
+    url: https://ssw.com.au/people/daniel-mackay
+related: []
+created: 2022-11-21T00:28:43.256Z
+guid: 4ff555fd-3f4c-416b-9fc3-1bf9409cc0ab
+---
+These days pull requests are the de facto standard for getting code reviewed.  Once a developer has finished their change, they will typically submit a pull request and move on to their next task.  This allows for an asynchronous process to take place which may seem like a good idea, but this can also lead to inefficiencies.
+
+## Inefficient Code Reviews
```

Comment:
Change heading to "Problem - Inefficient Code Reviews"

Then I would summarize the sub headings as bullet points to give the reader an idea of what they are in for

e.g.
Pull Requests can cause inefficient code review processes due to:
* Providing feedback too late
* Providing feedback too slow
* ...
* ...

Reply:
Updated